mods/sp/rules/vehicles.yaml

```diff
@@ -1725,31 +1623,19 @@ HVRTRUK3:
 		Range: 6c0
 		Type: ExplosionRange
 		Color: FFA500CC
-	Turreted:
-	AttackTurreted:
```

Comment:
Needs attack turreted for oportunity fire, attack frontal makes zero sense for a suicide unit.

Reply:
added facing tolerance to 128.